### ISSUE TITLE
Entering training stage

### ISSUE THREAD

**Issue by siemdejong** [siemdejong/skinstression#28]
We entered training stage. Below a TODO list laying out the steps to take before the milestone is reached.

**Prepare hyperparameter optimization**
- [x] #29
- [x] #30
- [x] #26 (but with imbalanced labels from a logistic curve fit)
- [x] #25
- [x] #21
- [x] #20
- [x] #18 (careful with information leakage)
- [x] #12
- [x] #9
- [x] #24 
- [x] Overfit using a small portion of the dataset
- [ ] Perform tests. Possibly implement dry-run functionality.
- [x] Perform hyperparameter optimization with train/test set (stratify on person id) #23 
- [ ] Wait...

**Prepare training**
- [ ] Implement Stratified K-fold cross validation (stratify on person id)
    - [x] #11
    - [ ] Rethink #6
- [ ] Wait...

**Cleanup**
- [ ] Celebrate (or not)
- [ ] Update documentation
